src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView/bpp.jelly

```diff
@@ -222,80 +222,67 @@
         <input name="triggerProjectName" id="triggerProjectName" type="hidden" />
 
         <div id="build-pipeline-plugin-content">
-            <div>
-                <h1>
-                    Build Pipeline<j:if test="${!empty(from.getBuildViewTitle())}">: ${from.getBuildViewTitle()}</j:if>
-                </h1>
-            </div>
+            <h1>Build Pipeline<j:if test="${!empty(from.getBuildViewTitle())}">: ${from.getBuildViewTitle()}</j:if></h1>
             <div>${from.getDescription()}</div>
 
-      <j:set var="buildPipelineForm" type="au.com.centrumsystems.hudson.plugin.buildpipeline.BuildPipelineForm" value="${from.getBuildPipelineForm()}" />
-      <j:set var="projectGrid" value="${buildPipelineForm.getProjectGrid()}" />
+            <j:set var="buildPipelineForm" type="au.com.centrumsystems.hudson.plugin.buildpipeline.BuildPipelineForm" value="${from.getBuildPipelineForm()}" />
+            <j:set var="projectGrid" value="${buildPipelineForm.getProjectGrid()}" />
 
             <div id="icon-bar">
                 <!-- If the user has the Build Project permission include a hyperlink to build the base project.-->
-        <j:set var="hasBuildPermission" type="boolean" value="${from.hasBuildPermission()}" />
-        <j:if test="${hasBuildPermission}">
-            <div class="icon-container">
-            <j:if test="${from.isProjectParameterized()}">
-                <span class="pointer" onclick="buildPipeline.fillDialog('${app.rootUrl}${from.gridBuilder.firstJobLink}/build?delay=0sec', 'Starts the pipeline with parameters')">
-                    <img src="${rootURL}/images/24x24/clock.png" alt="Trigger a Pipeline" class="icon-with-caption"/>
-                </span>
-            </j:if>
-            <j:if test="${!from.isProjectParameterized()}">
-              <a id="trigger-pipeline-button" href='#' onclick="$('triggerPipelineForm').submit()">
-                <img src="${rootURL}/images/24x24/clock.png" alt="Trigger a Pipeline" class="icon-with-caption"/>
-              </a>
-            </j:if>
-            <div>Run</div>
-          </div>
-        </j:if>
+                <j:set var="hasBuildPermission" type="boolean" value="${from.hasBuildPermission()}" />
+                <j:if test="${hasBuildPermission}">
+                    <div class="icon-container">
+                        <j:if test="${from.isProjectParameterized()}">
+                            <span class="pointer" onclick="buildPipeline.fillDialog('${app.rootUrl}${from.gridBuilder.firstJobLink}/build?delay=0sec', 'Starts the pipeline with parameters')">
+                                <img src="${rootURL}/images/24x24/clock.png" alt="Trigger a Pipeline" class="icon-with-caption"/>
+                                <span>Run</span>
```

Comment:
This is a bit hard to review: style changes (spacing) are mixed with content changes. In the future I'd like to avoid this, to make review easier and to preserve a good commit history. Perhaps this is something to add to the contributing rules.

With that said - it looks like the content change is just these <span> elements - is that correct? These were removed in an earlier commit?